functions/linkedInReports.js

```diff
@@ -26,7 +26,7 @@ router.post('/', async (request, response) => {
     } = request;
     const TableName = process.env.LINKED_IN_REPORTS_TABLE_NAME;
     const now = Date.now();
-    const Item = Object.assign({ id: now.toString(), createdAt: new Date(now).toISOString() }, payload);
```

Comment:
I don't like this idea, because we already have records submitted to a table. So we are not able to change ID column type now